scripts/consistent-deps.sh

```diff
@@ -16,11 +16,14 @@ function join_by {
 }
 
 # NOTE(alan): Add any dependency files here we don't want to include in the resolution.
-excludefiles=("requirements/build.txt")
+excludefiles=("requirements//build.txt")
 
 # Build an array of requirements files.
 shopt -s nullglob
-reqfiles=(requirements/*.txt)
+reqfiles=()
+while IFS= read -r -d $'\0'; do
```

Comment:
What does this change to the script do?

Reply:
Is this to get the ingest requirements from the subdirectory?

Reply:
This creates a bash array out of all the `*.txt` files in here. This was previously missing the ones in the ingest folder.

Reply:
Thanks!